android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -928,7 +928,9 @@ public void onDismiss() {
 
     dismissHelpBubble();
     //keyPreviewWindow.setAnimationStyle(R.style.KeyPreviewAnim);
-    keyPreviewWindow.showAtLocation(KMKeyboard.this, Gravity.TOP | Gravity.LEFT, posX, posY);
+    if (keyPreviewWindow != null) {
```

Comment:
In what situations would the preview window be null? Would that mean that in some circumstances the key preview window wouldn't display when it should?

Reply:
Crashlytics had 10 instances of this crash, and 90% of the time the app was in the background.
I suspect there's a race condition of 
```
    keyPreviewWindow.setOnDismissListener(new OnDismissListener() {
      @Override
      public void onDismiss() {
        keyPreviewWindow = null;
      }
    });
```
with a preview key being shown